src/main/resources/changesets/changelog-consent-104.0.xml

```diff
@@ -0,0 +1,31 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="104.0" author="connorlbark">
+        <createTable tableName="file_storage_object">
+            <column name="file_storage_object_id" autoIncrement="true" type="int">
+                <constraints primaryKey="true"/>
+            </column>
+            <!-- Could be: Reference ID, Dataset ID column, etc. -->
+            <column name="entity_id" type="text"/>
+            <column name="file_name" type="text" >
+                <constraints nullable="false"/>
+            </column>
+            <column name="gcs_file_uri" type="text">
+                <constraints nullable="false"/>
+            </column>
+            <column name="category" type="text"/>
+            <column name="media_type" type="text"/>
+            <column name="create_user_id" type="integer"/>
```

Comment:
Do we always require the user?  If so, nullable="false" should be a constraint.
Any thoughts on a foreign key constraint like you suggested in my changeset 103?

Reply:
added foreign keys to all of the user_ids!